### ISSUE TITLE
Add alt text to immage for the blind

### ISSUE THREAD

**Issue by HeWhoThreadsLightly** [AlexTelon/FlexiCharge-My-pages#171]
**Immage elements are missing alt text for the blind.

**Describe the solution you'd like**
Add alt text to immages like our logo so the blind can navigate on the webpage.
<img class="makeStyles-navLogo-10" src="/static/media/header-profile.a4b518b2.svg" alt="Flexi charge logo">

**Describe alternatives you've considered**
None

**Additional context**
[Add any other context or screenshots about the feature request here.](https://www.w3schools.com/tags/att_img_alt.asp)
